src/screens/ChannelListScreen/index.tsx

```diff
@@ -11,7 +11,6 @@ import GroupChatChannelItem from '../../components/ChatList/GroupChatChannelItem
 import MessageChannelItem from '../../components/AnonymousChat/MessageChannelItem';
 import PostNotificationChannelItem from '../../components/AnonymousChat/PostNotificationChannelItem';
 import Search from './elements/Search';
-import useFollowUser from './hooks/useFollowUser';
 import useLocalDatabaseHook from '../../database/hooks/useLocalDatabaseHook';
 import useRootChannelListHook from '../../hooks/screen/useRootChannelListHook';
 import useSignedChannelListScreenHook from '../../hooks/screen/useSignedChannelListHook';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `useFollowUser` hook has been removed and no new hook is imported in its place. Make sure that the functionality provided by this hook is either no longer needed or is being handled elsewhere.
```diff
- import useFollowUser from './hooks/useFollowUser';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->